src/samples/EducationStart/index.tsx

```diff
@@ -287,7 +288,7 @@ const EducationStartCase: FunctionComponent<any> = () => {
                 <div id='pega-root'></div>
               </div>
               {showLandingPage && (
-                <MainWrapper showPageNotWorkingLink={showLandingPage}>
+                <MainWrapper showPageNotWorkingLink={false} pageNotWorkingURL={pageNotWorkingURL}>
```

Comment:
You are passing the new URL but showPageNotWorkingLink is false. I am confused what you are trying to do it here.